app/listings/package.json

```diff
@@ -7,8 +7,8 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "axios": "^0.21.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^16.8.0",
```

Comment:
Is the version change necessary or its just due to your working station?

Reply:
It's necessary because of `react-tinder-card`. This package has a peer dependency for `react` version 16. This means that the package is compatible with react v16 and not v17. When `react` is set to version 17 in the `package.json` dependencies, the `npm install` command failes.